frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,168 @@
+import { execSync } from 'child_process';
+import { browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  MINUTE,
+  NS,
+  OCS_NODE_LABEL,
+  POD_NAME_PATTERNS,
+  STORAGE_CLASS_PATTERNS,
+  STORAGE_CLUSTER_NAME,
+} from '../utils/consts';
+import {
+  InstallCluster,
+  filterInput,
+  goToStorageClasses,
+  ocsOperatorStatus,
+} from '../views/installFlow.view';
+import {
+  checkIfClusterIsReady,
+  getDataFromRowAndCol,
+  getPodData,
+  getPodPhase,
+  podNameFilter,
+  sendKeys,
+  testPodIsRunning,
+  verifyNodeLabels,
+} from '../utils/helpers';
+
+const Installer = new InstallCluster(NS);
+
+describe('Testing OCS Subscription', () => {
+  beforeAll(async () => {
+    await Installer.createNamespace();
+    expect(browser.getCurrentUrl()).toContain(NS);
+  }, 2 * MINUTE);
+
+  it(
+    'tests subscription flow for ocs operator',
+    async () => {
+      await Installer.subscribeToOperator();
+      const text = await ocsOperatorStatus.getText();
+      expect(text.includes('Succeeded')).toBe(true);
+    },
+    2 * MINUTE,
+  );
+
+  it('tests for presence of 3 operator pods', async () => {
+    const podList = JSON.parse(
+      execSync('kubectl get po -n openshift-storage -o json').toString('utf-8'),
+    );
+    const pods = podList.items;
+    let pod = getPodData(pods, POD_NAME_PATTERNS.OCS);
+    let phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.ROOK);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.NOOBA_OPERATOR);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+  });
+});
+
+describe('Test creation of Storage Cluster', () => {
+  it(
+    'creates a storage cluster',
+    async () => {
+      const nodes = await Installer.createStorageCluster();
+      const text = await crudView.resourceTitle.getText();
+      expect(text).toEqual(STORAGE_CLUSTER_NAME);
+      // Wait for cluster to come to ready state
+      await checkIfClusterIsReady();
+      // Verify all the nodes have the required labels
+      nodes.forEach((node) => {
+        expect(verifyNodeLabels(node, OCS_NODE_LABEL)).toBe(true);
+      });
```

Comment:
Check for Node Labels before `checkIfClusterIsReady()`. Because, cluster will not be ready if nodes are not properly labelled.

Reply:
ack